app/src/org/commcare/views/widgets/RecordingFragment.java

```diff
@@ -74,6 +84,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return layout;
     }
 
+    private void initAudioFile() {
+        fileName = Environment.getExternalStorageDirectory().getAbsolutePath() + FILE_BASE + listener.getFileUniqueIdentifier() + FILE_EXT;

```

Comment:
If the generic landing point here is going to be "CommCare's storage" I think we should point it to the same 'temp' directory where we unzip CCZ files, since that directory is regularly purged.

Reply:
@ctsims  Agree, the `FILE_BASE` here is actually a path with value `/Android/data/org.commcare.dalvik/temp/Custom_Recording` that takes care of this.